<?xml version="1.0" ?>
<launch>
  <arg name="robot_ip" />
  <arg name="load_gripper" default="false" />
  <include file="$(find franka_control)/launch/franka_control.launch" >
    <arg name="robot_ip" value="$(arg robot_ip)" />
    <arg name="load_gripper" value="$(arg load_gripper)" />
  </include>

  <include file="$(find qb_hand_control)/launch/control_qbhand.launch" >
    <arg name="standalone" value="true" />
    <arg name="activate_on_initialization" value="true" />
    <arg name="use_waypoints" value="false" />
    <arg name="device_id" value="2" />
  </include>

  <node pkg="franka_tool_handover" name="JointAS_rec" type="node"/>
  <rosparam command="load" file="$(find franka_tool_handover)/config/franka_handover_controllers.yaml" />
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"  args="joint_impedance_controller"/>
</launch>
